src/main/java/filemanager/model/XmlModel.java

```diff
@@ -1,13 +1,32 @@
 package filemanager.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+
+import javax.xml.bind.annotation.XmlRootElement;
 import java.util.ArrayList;
 
+@XmlRootElement
 public class XmlModel {
 
+    @JsonProperty("client")
+    @JacksonXmlProperty(localName = "clientName")
```

Comment:
Do you really need to provide ```localName``` for properties which name matches to name element? I've also just remembered that xml elements should actually start from Capital letters. You can find our PPE feed schema here http://release-assets.bazaarvoice.com/xs/PRR/PostPurchaseFeed/14.7 (it is a bit complicated to read, but you can easily find field names there)